docs/dev-docs/monitoring-and-alerting/configure-monitoring-backend.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+## Monitoring of Real-Time Routers
+
+Turing routers (as well as enrichers and ensemblers) are deployed using the Knative framework, on top of the Istio Service mesh. Both tools provide some out-of-the box Prometheus metrics to track common stats such as error rate, request latency, etc. and can be configured by following the official guides:
+* [Knative](https://knative.dev/docs/serving/observability/metrics/serving-metrics/#autoscaler)
+* [Istio](https://istio.io/v1.9/docs/tasks/observability/metrics/)
+
+The following custom Prometheus metrics are published by the Turing routers too.
+
+| Metric Name | Description | Type | Tags | Unit |
+| ----------- | ----------- | ---- | ---- | ---- |
+| mlp_turing_exp_engine_request_duration_ms | The duration for fetching a treatment from the experiment engine | Histogram | `status`, `engine` | Milliseconds |
+| mlp_route_request_duration_ms | The duration for the call to a route | Histogram | `status`, `route` | Milliseconds |
+| mlp_turing_comp_request_duration_ms | The duration for a custom operation in the code, useful for debugging | Histogram | `status`, `component` | | Milliseconds |
```

Comment:
I might be saying something silly but isn't it meant to be '*complete* operation' and not 'custom operation'?

Reply:
It is supposed to be "custom" - i.e., some intermediate steps in the request flow (eg: write outcome log).